### DISCUSSION TITLE
When admin account create user, it can select organization

### DISCUSSION THREAD

**Discussion by YuanXingH** [grafana/grafana#42862] [General]
Now, when admin account create user, it can not select organization.  The user is under Main Org by default

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
